packages/rum/test/benchmarks/analyzer.js

```diff
@@ -130,9 +160,20 @@ function calculateResults(resultMap) {
     Object.keys(metricObj).forEach(metricName => {
       const value = metricObj[metricName]
       /**
-       * deal with common data points
+       * Add consumed memory in bytes per each function to the result
        */
-      if (!Array.isArray(value)) {
+      if (metricName === 'memory') {
+        const reducedValue = value.reduce((acc, curr) => {
+          acc.push(...curr)
+          return acc
+        }, [])
+        reducedValue.forEach(obj => {
+          result[`memory.${obj.name}.bytes`] = obj.size
```

Comment:
This field will be parsed in Elasticsearch into hierarchy of objects, is that right?